extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java

```diff
@@ -499,7 +488,7 @@ public ServletDeploymentManagerBuildItem build(List<ServletBuildItem> servlets,
             String filterClass = filter.getFilterClass();
             reflectiveClasses.accept(new ReflectiveClassBuildItem(false, false, filterClass));
             recorder.registerFilter(deployment, filter.getName(), context.classProxy(filterClass), filter.isAsyncSupported(),
-                    bc.getValue(), filter.getInitParams(), filter.getInstanceFactory());
+                    bc.getValue(), filter.getInstanceFactory());
```

Comment:
Where are these init params used?

Reply:
@gastaldi 
Well, they were redundantly used in in ```io.quarkus.undertow.runtime.UndertowDeploymentRecorder#registerFilter```.
The real place where initParams are set for filters is this: [UndertowBuildStep.java#L406](https://github.com/quarkusio/quarkus/blob/af0251cfff43b386814f5a44285dd515810996a8/extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java#L406).